src/main.go

```diff
@@ -364,6 +370,30 @@ func (t *tester) Run() error {
 				q.Query = q.Query[:len(q.Query)-1]
 			}
 			t.disconnect(q.Query)
+		case Q_LET:
+			q.Query = strings.TrimSpace(q.Query)
+			eqIdx := strings.Index(q.Query, "=")
+			if eqIdx > 1 {
+				start := 0
+				if q.Query[0] == '$' {
+					start = 1
+				}
+				varName := q.Query[start:eqIdx]
```

Comment:
Should not `varName` also be trimmed, if there is a space between the variable name and the equal sign?
```suggestion
				varName := strings.TrimSpace(q.Query[start:eqIdx])
```